packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -95,110 +46,97 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
         break;
     }
 
-    // contruct a cloud of points (nodes of a structured grid)
-    auto cloud = make_stuctured_cloud( Lx, Ly, Lz, nx, ny, nz );
-    int n = cloud.size();
-
-    Kokkos::View<DataTransferKit::Box *, DeviceType> bounding_boxes(
-        "bounding_boxes", n );
-    auto bounding_boxes_host = Kokkos::create_mirror_view( bounding_boxes );
-    // build bounding volume hierarchy
-    for ( int i = 0; i < n; ++i )
+    Kokkos::View<DataTransferKit::Point *, DeviceType> random_points(
+        "random_points" );
     {
-        auto const &point = cloud[i];
-        double x = std::get<0>( point );
-        double y = std::get<1>( point );
-        double z = std::get<2>( point );
-        bounding_boxes_host[i] = {{{x, y, z}}, {{x, y, z}}};
+        auto n = std::max( n_values, n_queries );
+        Kokkos::resize( random_points, n );
+
+        auto random_points_host = Kokkos::create_mirror_view( random_points );
+
+        // edge length chosen such that object density will remain constant as
+        // problem size is changed
+        auto const a = std::cbrt( n_values );
```

Comment:
Should you also allow for 2D problems in this test? This would require changing `cbrt` to `sqrt` and some other things in it.